packages/opine/opine.0.1.2/opam

```diff
@@ -0,0 +1,51 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Python AST unparse implementation in OCaml"
+description:
+  "Opine is an OCaml library that unparses the python AST back to python code."
+maintainer: ["arulselvan1234@gmail.com"]
+authors: ["Arulselvan Madhavan"]
+license: "MIT"
```

Comment:
There doesn't seem to be any license info in your repo (no `LICENSE` file and no license headers). You should probably add some info over there too.

Reply:
Added a LICENSE file